monkey/infection_monkey/credential_collectors/credential_types.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class CredentialTypes(Enum):
```

Comment:
By convention, Enum names are singular.

```suggestion
class CredentialType(Enum):
```